docs/setup/developer.md

```diff
@@ -1,21 +1,54 @@
 # Developer setup
 
+## Installation
+
+You can install all PIXL Python modules by running the following commands from the `PIXL/` directory:
+
+```shell
+python -m pip install -e "pixl_core/[dev]"
+python -m pip install -e "pytest-pixl/[dev,test]"
+python -m pip install -e "pixl_core/[test]"
```

Comment:
this seems convoluted but is necessary as `pixl_core[test]` depends on `pytest_pixl`, and `pytest_pixl[test]` depends on `pixl_core`

Reply:
yeah agreed, its fairly painful because we're using core in pytest pixl